src/python/pants/backend/helm/subsystems/post_renderer.py

```diff
@@ -171,7 +207,19 @@ async def setup_post_renderer_launcher(
     )
 
     # Build a shell wrapper script which will be the actual entry-point sent to Helm as the post-renderer.
-    post_renderer_process_cli = " ".join(post_renderer_process.argv)
+    # Extra post-renderers are plugged by piping the output of one into the next one in the order they
+    # have been defined.
+    extra_post_renderers = (
+        await _resolve_post_renderers(request.extra_post_renderers)
+        if request.extra_post_renderers
+        else []
+    )
+    post_renderer_process_cli = " | ".join(
+        [
+            " ".join(post_renderer_process.argv),
+            *[" ".join(post_renderer.args) for post_renderer in extra_post_renderers],
+        ]
+    )
```

Comment:
Does this need any kind of shell escaping?

Reply:
Good point. Made a bit of a lame approach using `re.sub(...)` to force single quotes around each individual argument although I don't think I got the expression right... It would be great if someone that is better at me at translating `sed` commands into Python could take a look at it.

I took it from this SO post:
https://stackoverflow.com/questions/15783701/which-characters-need-to-be-escaped-when-using-bash

Reply:
mmm, wanted to avoid taking a new dependency but will consider it